backend/src/routes/index.ts

```diff
@@ -220,6 +246,7 @@ const routes: FastifyPluginAsync = async (server) => {
           if (!(TOKEN_ADDRESS[chainId] && TOKEN_ADDRESS[chainId][gasToken])) return reply.code(ReturnCode.FAILURE).send({ error: "Invalid network/token" })
           result = await paymaster.pimlicoAddress(gasToken, networkConfig.bundler, entryPoint);
```

Comment:
Remember to catch here, do not assume that these calls will always be successful

Reply:
we have a try/catch block for this entire request so if something goes wrong it responds that something is wrong and we have logs to check it out

Reply:
Remember that "something is wrong" is simply an unacceptable error message. See https://github.com/etherspot/arka/pull/65/files#diff-7eb9e5d1a1dab305049dc4a305507fceb03a8f37aa37c7a778826d85b5a40748R254